ipn/ipnlocal/local.go

```diff
@@ -7489,3 +7498,42 @@ func maybeUsernameOf(actor ipnauth.Actor) string {
 	}
 	return username
 }
+
+// VIPServices returns the list of tailnet services that this node
+// is serving as a destination for.
+// The returned memory is owned by the caller.
+func (b *LocalBackend) VIPServices() []*tailcfg.VIPService {
+	b.mu.Lock()
+	defer b.mu.Unlock()
+	return b.vipServicesFromPrefs(b.pm.CurrentPrefs())
+}
+
+func (b *LocalBackend) vipServicesFromPrefs(prefs ipn.PrefsView) []*tailcfg.VIPService {
```

Comment:
This receiver is never used. Remove it (change it to a func instead of method), so then you don't need to worry about documenting whether this is a "Locked" method or not?